webpay/settings/base.py

```diff
@@ -27,6 +27,8 @@
 DEBUG = True
 DEBUG_PROPAGATE_EXCEPTIONS = True
 
+USE_TZ = True
```

Comment:
Yeah, it's a standard Django thing and I was surprised we didn't already have it. It makes date handling always use UTC. We don't use a database so it doesn't affect much. I grepped and couldn't really find anything historic that it would affect.